### PR TITLE
Update Treasury hook to 1.1.0.

### LINKED ISSUE #50: Update Treasury Integration
Treasury v1.1.0 has released which changes the way that economy providers are accessed - instead of using Bukkit's service manager, plugins should now use Treasury's service API to access it. Let me know if you need a hand with it 😃

### PR DESCRIPTION
**Pull request by RezzedUp**
Use Treasury's service registry.
Closes #50.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -115,7 +115,9 @@
         <dependency>
             <groupId>com.github.lokka30.treasury</groupId>
             <artifactId>treasury-api</artifactId>
-            <version>1.0.0</version>
+            <!-- Force a specific commit since they created the 1.1.0 git tag days before -->
+            <!-- they actually released 1.1.0... so the existing JitPack build for 1.1.0 is incorrect -->
+            <version>937ec97a98</version> <!-- (1.1.0-937ec97-RELEASE) -->
```
Comment (lokka30) [inline pom.xml:120]: I just told JitPack to scrap the current version it has built. Does 1.1.0 work in pom now? :)

**Review by lokka30 — CHANGES_REQUESTED** [review]
Cheers Rezz!

File: pom.xml (hunk 1 above)
Comment (RezzedUp) [reply pom.xml:120]: No, because the `1.1.0` build is still from February 14: https://jitpack.io/com/github/lokka30/treasury/1.1.0/build.log

Comment (lokka30) [reply pom.xml:120]: JitPack is being weird. I can delete the old 1.1 builds, but they still seem to be used by our IDEs. And using this link instead, which just has a capital T for Treasury, yields the latest build for some reason.

https://jitpack.io/com/github/lokka30/Treasury/1.1.0/build.log

**Pull request merged**